src/DevChatter.Bot/Startup/SetUpDatabase.cs

```diff
@@ -65,13 +65,61 @@ private static void EnsureInitialData(IRepository repository)
                 repository.Create(GetInitialQuizQuestions());
             }
 
+            CreateInitialRouletteSettingsIfNeeded(repository);
+
             var missingCommandWords = GetMissingCommandWords(repository);
             if (missingCommandWords.Any())
             {
                 repository.Create(missingCommandWords);
             }
         }
 
+        private static void CreateInitialRouletteSettingsIfNeeded(IRepository repository)
+        {
+            var commandName = "roulette";
```

Comment:
We should make sure that this uses the CommandWord from Roulette. If translated (or changed), it could be different.